src/test/kotlin/org/rust/ide/inspections/lints/naming/RsVariableNamingInspectionTest.kt

```diff
@@ -61,7 +61,7 @@ class RsVariableNamingInspectionTest : RsInspectionsTestBase(RsVariableNamingIns
             let (real, imaginary) = (7.2, 3.5);
             println!("{} + {}i", real, imaginary);
         }
-    """)
+    """, previewExpected = false)
```

Comment:
Are we going to make `RenameFix` work in preview?

Reply:
It is possible in future PRs, `RenameFix` is listed in corresponding section in #9561